test/natalie/break_test.rb

```diff
@@ -176,4 +176,16 @@ def break_in_begin_in_while
     result = break_in_begin_in_while { |x| break :ok }
     result.should == :ok
   end
+
+  fit 'should not leak break flag' do
```

Comment:
focused test :joy:

Reply:
Well... Seems like my self-protection measure did not work 😅 Will fix those tomorrow!

Reply:
I think we need to pass the CI env variable to the docker container stuff somehow to make the "focus"-detection work?

Reply:
Yep! I fixed that in 81027a4bf73755256078de21556627678ef7034b just now. :hugs: